app/components/finance/npq/participant_outcomes/table.rb

```diff
@@ -16,16 +14,61 @@ def render?
           participant_declaration.participant_profile.npq? && outcomes.present?
         end
 
+        def state(outcome)
+          outcome.state&.capitalize
+        end
+
         def completion_date(outcome)
           outcome.completion_date.to_fs(:govuk)
         end
 
         def changed_date(outcome)
-          outcome.created_at.to_fs(:govuk)
+          outcome.created_at.to_date.to_fs(:govuk)
+        end
+
+        def sent_to_tra_at(outcome)
+          return tag.strong(t("finance.npq.participant_outcomes.na"), class: "govuk-tag govuk-tag--yellow") if outcome.sent_to_qualified_teachers_api_at.blank?
```

Comment:
I'm not sure about the tag being yellow here, as in the prototype its only on `Recorded by TRA API`, but let's see what PO review says and leave as is for now :)